wgengine/router/router_linux.go

```diff
@@ -579,11 +529,19 @@ func (r *linuxRouter) setNetfilterMode(mode preftype.NetfilterMode) error {
 	return nil
 }
 
+func (r *linuxRouter) getV6Available() bool {
+	return r.nfr.GetV6Available()
```

Comment:
Why is this its own method? could the caller just do `r.nfr.GetV6Available()` themselves?

Reply:
Thanks for the explanation!